tests/fixed_vector_test.cpp

```diff
@@ -0,0 +1,131 @@
+#define CATCH_CONFIG_MAIN
+#include <catch2/catch.hpp>
+
+#include <nitro/lang/fixed_vector.hpp>
+
+TEST_CASE("fixed vector init", "[lang]")
+{
+    nitro::lang::fixed_vector<std::int64_t> v(10);
```

Comment:
Use constexpr variables for literal constants

Reply:
To make fixed_vector able for constexpr i need CppStd 20 (like the std::vector too)

Reply:
Now I see what you were talking about with the constexpr. I may have sent you on a different track here. What my original intent was, was for you to replace the number literals in the tests with a const (or constexpr) variable.

In this particular case:

```suggestion
    constexpr int SIZE = 10;
    //....
    nitro::lang::fixed_vector<std::int64_t> v(SIZE);
    // ...
    REQUIRE(v.capacity() == SIZE);
```